api/goldens/aptos_api__tests__transactions_test__test_gas_estimation_disabled.json

```diff
@@ -0,0 +1,5 @@
+{
+  "deprioritized_gas_estimate": 0,
+  "gas_estimate": 0,
```

Comment:
So this is how it worked before I suppose? I recall that in our test environment the gas unit price is zero, is that why this estimate is zero? Because in reality if we're returning `gas_estimate = 0` that would be really bad.

Reply:
Yup only expected in the test environment because of https://github.com/aptos-labs/aptos-core/blob/1c5f49514116d545f7cd3efe2a37204da4ee07da/config/global-constants/src/lib.rs#L25

Reply:
Okay all good then!

Reply:
For later on: https://github.com/aptos-labs/aptos-core/issues/8293.